wmt-web/README.md

```diff
@@ -0,0 +1,16 @@
+# WMT Web
+This is the webserver application that will present the information stored in
+the WMT database to the end user.
+
+It is a node.js application using the express web framework.
+
+## Prerequisites
+- Node v7.6.x
```

Comment:
Good point. I've checked on nvm and the most up-to-date LTS appears to be 6.10.0, happy if we use that from here on out?